middleware/2023-09-06-1.2.21103.40594.md

```diff
@@ -0,0 +1,29 @@
+---
+authors: poscreator
+slug: middleware/1.2.21103.40594
+tags: [Middleware, France]
+---
+
+# Middleware 1.2.21103.40594 SignaturCloud (France)
+In this version of the French Middleware, we've improved the performance of the copy receipt.
+
+<!--truncate-->
+
+:::caution
+
+Version 1.2 of the Middleware is meant for the Austrian and French market only, customers in Germany have to use version 1.3. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: Improved the performance of the copy receipt
+We've improved the performance of the French Middleware when dealing with copy receipts by counting the copy receipts through the journalfr base64 decoded payload, for the SignaturCloud.
+More details can be found in on [Github](https://github.com/fiskaltrust/middleware/blob/main/rfcs/0192-fr-copy-receipt-performance.md)
+
+### Affected packages:
+- _fiskaltrust.service.azure v1.2.21103.40594_
```

Comment:
```suggestion
- _fiskaltrust.service.azure v1.2.23219.65461-rc1_
```

The azure package is not deployed to portal production though (do we need that? I think its only used by the signaturecloud)